index.js

```diff
@@ -153,17 +154,34 @@ function renderUserCart(){
         item.quantity -= 1
       }
       renderUserCart()
+      renderTotalPrice()
     })
 
     addButton.addEventListener("click", function() {
       item.quantity += 1
       renderUserCart()
+      renderTotalPrice()
     })
     }
 }
 
+function renderTotalPrice() {
+  const currencyFormatter = new Intl.NumberFormat('en-GB', {
+    style: 'currency',
+    currency: 'GBP',
+  });
+
+  totalPrice = 0
+  for (item of state.cart) {
+    totalPrice += item.price * item.quantity
+  }
+  totalPriceContainer = document.querySelector('.total-number')
+  totalPriceContainer.innerText = currencyFormatter.format(totalPrice)
+}
+
 function init() {
   renderStoreIcons();
+  renderTotalPrice();
 }
 
 init();
```

Comment:
What is this function for? seems interesting

Reply:
init() just starts up the program, as that is the only function that is called when the programme runs, so everything starts from here. It could have been called anything, I just called it "init" to mean "initialize".

When init() is called, it starts up rendering the store icons and the total price by calling renderStoreIcons() and renderTotalPrice().